testah-junit/src/main/java/org/testah/framework/annotations/TestCaseJUnit5.java

```diff
@@ -0,0 +1,105 @@
+package org.testah.framework.annotations;
+
+import org.junit.jupiter.api.Test;
+import org.testah.client.enums.TestType;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * The Interface TestCase.
+ */
+@Inherited
+@Retention(RetentionPolicy.RUNTIME)
+@Test
+@TestCase
+@Target({ElementType.ANNOTATION_TYPE,ElementType.METHOD})
+public @interface TestCaseJUnit5 {
+
+    /**
+     * Id.
+     *
+     * @return the int
+     */
+    int id()
+
+            default -1;
+
+    /**
+     * Name.
+     *
+     * @return the string
+     */
+    String name()
+
+            default "";
+
```

Comment:
do you have to have default values for these 2?

Reply:
yes, it would be a breaking change to not.